edx_proctoring/api.py

```diff
@@ -1490,6 +1490,10 @@ def _get_timed_exam_view(exam, context, exam_id, user_id, course_id):
             )
 
         total_time = humanized_time(allowed_time_limit_mins)
+
+        # According to WCAG, there is no need to allow for extra time if > 20 hours allowed
+        hide_extra_time_footer = exam['time_limit_mins'] > 20 * 60
```

Comment:
Haha, I had the same thought. It's defined as ["longer than 20 hours"](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-required-behaviors.html)